Readme.md

```diff
@@ -1,31 +1,42 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/230140164/20.2.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T848300)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Index.razor](./CS/BlazorProject/Pages/Index.razor)
-<!-- default file list end -->
+# DataGrid for Blazor - How to create the DataGrid and its columns at runtime
+
+This example shows how to create the Data Grid component at runtime and generate its columns dynamically.
 
-### DataGrid for Blazor - How to create DataGrid and its columns at runtime
-<!-- run online -->
-**[[Run Online]](https://codecentral.devexpress.com/230140164/)**
-<!-- run online end -->
+![Change Grid Content at Runtime](images/datagrid-change-data-on-run.gif)
+
+The main idea of this example is to implement a method that returns the **RenderFragment<T>** object, for example, as follows:
```

Comment:
It seems strange to say that this example shows how to do something, and then say that there is another main idea. So what is this example about exactly then?

Reply:
To do this, implement...